### ISSUE TITLE
Add a new tag to the old data

### ISSUE THREAD

**Issue by Misenko** [OpenTSDB/opentsdb#705]
Hi,

I’ve been using OpenTSDB for a half a year now and I was wondering if it is somehow possible to add a new tag to the old data. I know that I can do `scan –import –delete` and then do `import` but the data in its plain text form would take so much space that it’s not really an option for me. What I want to do is for every data that has tag-value pair `xyz=123` add a new tag-value pair `uvw=456`. Is there a way to do this via your API? Or maybe directly in HBase? I don’t mind doing some coding if you can point me in the right direction.

**Comment by manolama** [comment]
Yeah this would be a new feature and require a bit of coding. Not too much though. What you'll want to work off https://github.com/OpenTSDB/opentsdb/blob/master/src/tools/DumpSeries.java#L99-L149 and where it's reading the row, you'll need to build a new row key after resolving the UIDs, then make a ``putRequest`` with the new key and current values as well as a ``deleteRequest`` with the old row key that's a callback after the putRequest is complete (so as not to lose any data). Be careful to check the key so that if already has the new combo that it's not re-written/delete. Also make sure the keys are sorted properly.
